include/mpi_comm_handle.h

```diff
@@ -0,0 +1,11 @@
+#ifndef _COMM_HANDLE_H
+#define _COMM_HANDLE_H
+
+#include <mpi.h>
+extern MPI_Comm MPI_COMM_HANDLE;
+
+extern "C"{
```

Comment:
This needs some `#ifdef` to make sure it builds for MPI, QMP, and single GPU. Currently the MPI based CI test fail as they don't use QMP.